cibuildwheel/util.py

```diff
@@ -121,7 +123,42 @@ def __repr__(self):
         return f'{self.__class__.__name__}{self.base_file_path!r})'
 
 
+class Architecture(str, Enum):
+    # mac/linux archs
+    x86_64 = 'x86_64'
+    i686 = 'i686'
+    aarch64 = 'aarch64'
+    ppc64le = 'ppc64le'
+    s390x = 's390x'
+
+    # windows archs
+    x86 = 'x86'
+    amd64 = 'AMD64'
```

Comment:
Shouldn’t the caps match here?

Reply:
Though maybe it’s fine. Just want med to point out a potential confusing point.

Reply:
I think I agree with you, actually. Seems like it's asking for trouble, as written here